framework/CodeInterfaces/PHISICS/XSCreator.py

```diff
@@ -0,0 +1,177 @@
+"""
+Created on September 1st, 2017
+@author: rouxpn
+"""
+from __future__ import division, print_function, unicode_literals, absolute_import
+import warnings
+warnings.simplefilter('default',DeprecationWarning)
+import os
+from decimal import Decimal
+import xml.etree.ElementTree as ET
+from xml.etree.ElementTree import Element, SubElement, Comment
+from xml.dom import minidom
+
+class XSCreator():
+
+  def tabMapping(self,tab):
+    """
+      Links the tabulation number to the actual tabulation points
+      @ In, tab, string, refers to the tabulation number
+      @ Out, tabList, list, lists of all the tabulation parameters
+      @ Out, valueList, lists of all the tabulation values
+    """
+    mappingTree = ET.parse('tabMapping.xml')
```

Comment:
fixed. 
Consequently, I changed in the phisicsInterface the following: 
- the variable self.tabulation does not exist anymore. The user used to activate this flag through a readMoreXML node. 
- instead, if the type attribute 'tabMap' is found, automatically the tab mapping boolean variable is considered true, false otherwise.